stories/topics/proc-aws-backup-platform-stack.adoc

```diff
@@ -6,6 +6,16 @@
 . Pull the `ansible-on-clouds-ops 2.2` container image.
 +
 [NOTE]
+====
+Before Pulling the docker image, make sure you are login to redhat registry using docker. Use the below command to login to redhat registry. 
```

Comment:
Will need to fix all the additions that have "make sure you are login to redhat registry"
to either
"make sure you login to redhat registry"
OR
"make sure you are logged in to redhat registry"
@ianf77 @ariordan-redhat can help with what text they want to use